website/docs/tutorials/sharepoint.md

```diff
@@ -46,6 +46,98 @@ Let's walk through how OpenConnector can automatically publish WOO documents fro
 
 This automated flow ensures that WOO documents are consistently and accurately published while maintaining security and compliance requirements.
 
+## How to configure SharePoint authentication
+
+### Prerequisites
+- A Microsoft SharePoint instance managed with Microsoft Entra.
+- A Nextcloud instance with OpenConnector installed.
+- A valid X.509 certificate and the private key.
+- A Linux command line (or Git Bash on Windows) for generating values later in the process.
+
+### Setting up Microsoft Entra
+
+#### Create an app
+First, we need to register an appcreate an app registration in Microsoft Entra under ‘Identity’ → ‘Applications’ → ‘App registrations’. Give your app registration an appropriate name
+![](images/Entra_AppRegistration.png)
```

Comment:
Might need a newline here
![image](https://github.com/user-attachments/assets/474fbe15-f013-498e-9194-4bcd41fb1c8e)
Also image is small and not very readable like this